deploy/olm-catalog/kogito-operator/manifests/kogito-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1352 @@
+              "envs": [
+                {
+                  "name": "CI",
+                  "value": "true"
+                }
+              ],
+              "gitSource": {
+                "contextDir": "process-springboot-example",
+                "reference": "master",
+                "uri": "https://github.com/kiegroup/kogito-examples"
+              },
+              "incremental": true
+            },
+            "envs": [
+              {
+                "name": "DEBUG",
+                "value": "true"
+              }
+            ],
+            "runtime": "springboot"
+          }
+        },
+        {
+          "apiVersion": "app.kiegroup.org/v1alpha1",
+          "kind": "KogitoApp",
+          "metadata": {
+            "name": "kogito-travel-agency"
+          },
+          "spec": {
+            "build": {
+              "gitSource": {
+                "contextDir": "06-kogito-travel-agency",
+                "uri": "https://github.com/kiegroup/kogito-travel-agency-tutorial/"
+              }
+            },
+            "enableEvents": true,
+            "enablePersistence": true,
+            "envs": [
+              {
+                "name": "MP_MESSAGING_INCOMING_VISASAPPROVED_BOOTSTRAP_SERVERS"
+              },
+              {
+                "name": "MP_MESSAGING_INCOMING_VISASREJECTED_BOOTSTRAP_SERVERS"
+              },
+              {
+                "name": "MP_MESSAGING_OUTGOING_VISAAPPLICATIONS_BOOTSTRAP_SERVERS"
+              },
+              {
+                "name": "MP_MESSAGING_OUTGOING_KOGITO_PROCESSINSTANCES_EVENTS_BOOTSTRAP_SERVERS"
+              },
+              {
+                "name": "MP_MESSAGING_OUTGOING_KOGITO_USERTASKINSTANCES_EVENTS_BOOTSTRAP_SERVERS"
+              }
+            ]
+          }
+        },
+        {
+          "apiVersion": "app.kiegroup.org/v1alpha1",
+          "kind": "KogitoApp",
+          "metadata": {
+            "name": "kogito-visas"
+          },
+          "spec": {
+            "build": {
+              "gitSource": {
+                "contextDir": "06-kogito-visas",
+                "uri": "https://github.com/kiegroup/kogito-travel-agency-tutorial/"
+              }
+            },
+            "enableEvents": true,
+            "enablePersistence": true,
+            "envs": [
+              {
+                "name": "MP_MESSAGING_INCOMING_VISAAPPLICATIONS_BOOTSTRAP_SERVERS"
+              },
+              {
+                "name": "MP_MESSAGING_OUTGOING_VISASAPPROVED_BOOTSTRAP_SERVERS"
+              },
+              {
+                "name": "MP_MESSAGING_OUTGOING_VISASREJECTED_BOOTSTRAP_SERVERS"
+              },
+              {
+                "name": "MP_MESSAGING_OUTGOING_KOGITO_PROCESSINSTANCES_EVENTS_BOOTSTRAP_SERVERS"
+              },
+              {
+                "name": "MP_MESSAGING_OUTGOING_KOGITO_USERTASKINSTANCES_EVENTS_BOOTSTRAP_SERVERS"
+              }
+            ]
+          }
+        }
+      ]
```

Comment:
Please remove the `deploy/examples` directory, move it to the parent directory. On this new version, the SDK is considering every yaml file on `deploy` to be an example CR.